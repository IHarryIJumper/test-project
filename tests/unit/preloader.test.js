import React from 'react';
import renderer from 'react-test-renderer';
import PreloaderComponent from '../../client/src/scripts/components/preloader/preloader.jsx';


describe('Preloader matching schema', function () {
	test('without props', () => {
		const preloader = renderer.create(<PreloaderComponent />).toJSON();
		expect(preloader).toMatchSnapshot();
	});

	test('with show=true', () => {
		const preloader = renderer.create(<PreloaderComponent show={true} />).toJSON();
		expect(preloader).toMatchSnapshot();
	});

	test('with show=false', () => {
		const preloader = renderer.create(<PreloaderComponent show={false} />).toJSON();
		expect(preloader).toMatchSnapshot();
	});

});