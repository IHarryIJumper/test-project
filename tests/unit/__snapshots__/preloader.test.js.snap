// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Preloader matching schema with show=false 1`] = `null`;

exports[`Preloader matching schema with show=true 1`] = `
<div
  className="preloader-container"
>
  <div
    className="preloader"
  />
</div>
`;

exports[`Preloader matching schema without props 1`] = `
<div
  className="preloader-container"
>
  <div
    className="preloader"
  />
</div>
`;
